_uw-research-computing/htc-job-file-transfer.md

```diff
@@ -8,50 +8,65 @@ guide:
         - htc
 ---
 
+## Introduction
+
+This guide covers general information on using and transferring data on the HTC system. We will introduce you to the two file systems, how to determine which one is the best place for your data, and how to edit your submit file to transfer input and output files.
+
 {% capture content %}
-- [Data Storage Locations](#data-storage-locations)
-   * [/home](#home)
-   * [/staging](#staging)
-- [Transferring Data to Jobs with `transfer_input_files`](#transferring-data-to-jobs-with-transfer_input_files)
-   * [Important Note: File Transfers and Caching with `osdf:///`](#important-note-file-transfers-and-caching-with-osdf)
-- [Transferring Data Back from Jobs to `/home` or `/staging`](#transferring-data-back-from-jobs-to-home-or-staging)
-   * [Default Behavior for Transferring Output Files](#default-behavior-for-transferring-output-files)
-   * [Specify Which Output Files to Transfer with `transfer_output_files` and `transfer_output_remaps`](#specify-which-output-files-to-transfer-with-transfer_output_files-and-transfer_output_remaps)
+- [Introduction](#introduction)
+- [Data storage locations](#data-storage-locations)
+- [Transfer input data to jobs with `transfer_input_files`](#transfer-input-data-to-jobs-with-transfer_input_files)
+- [Transfer output data from jobs](#transfer-output-data-from-jobs)
+   * [Default behavior for transferring output files](#default-behavior-for-transferring-output-files)
+   * [Specify which output files to transfer with `transfer_output_files`](#specify-which-output-files-to-transfer-with-transfer_output_files)
+   * [Transfer files to other locations with `transfer_output_remaps`](#transfer-files-to-other-locations-with-transfer_output_remaps)
+   * [Transfer files to other locations with `output_destination`](#transfer-files-to-other-locations-with-output_destination)
+- [Transfer and unpack files with `osdf:///`](#transfer-and-unpack-files-with-osdf)
 - [Related pages](#related-pages)
 {% endcapture %}
 {% include /components/directory.html title="Table of Contents" %}
 
-## Data Storage Locations
+## Data storage locations
 
 <p style="text-align:center"><img src="/images/htc-data-spaces.png" width=300px></p>
 
 The HTC system has two primary locations where users can place their files:
 
-### /home
-* The default location for files and job submission
-* Efficiently handles many files
-* Smaller input files (<100 MB) should be placed here
-
-### /staging
-* Expandable storage system but cannot efficiently handle many small (few MB or less) files
-* Larger input files (>100 MB) should be placed here, including container images (.sif)
+|  | **`/home`** |  **`/staging`** |
+| ----------- | ----------- |
+| Purpose | Default file system, handles most files | Stages large files/containers for file transfer into jobs |
+| Job submission   | ✓ | ✕ | 
```

Comment:
"Job submission" is a little confusing. Not sure if I have an improvement. "Where to run condor_submit"? "Job submission command"?